game/campaignloader/defaultsquadronassigner.py

```diff
@@ -74,7 +74,7 @@ def find_squadron_for(
         # If we can't find any squadron matching the requirement, we should
         # create one.
         return self.air_wing.squadron_def_generator.generate_for_task(
-            config.primary, control_point
+            config.primary, control_point, self.game.settings.squadron_random_chance
```

Comment:
We can get rid of this parameter since `generate_for_task` has an indirect way of accessing the settings

Reply:
done